workers/unity/Assets/Fps/Scripts/GameLogic/StateMachine/Session/SessionState.cs

```diff
@@ -0,0 +1,20 @@
+namespace Fps
+{
+    public abstract class SessionState : State
+    {
+        protected readonly UIManager Manager;
+        protected readonly ConnectionStateMachine Owner;
+
+        protected SessionState(UIManager manager, ConnectionStateMachine owner)
+        {
+            Manager = manager;
+            Owner = owner;
+        }
+
+        protected void ShowErrorMessage(string errorMessage)
+        {
+            Manager.ScreenManager.StartScreenManager.ShowFailedToGetDeploymentsText(errorMessage);
+            Manager.ScreenManager.LobbyScreenManager.ShowFailedToGetDeploymentsText(errorMessage);
```

Comment:
because in `GetPlayerIdentityTokenState` we don't know in which screen we are currently in. It could be used for either the Start or the Lobby Screen. but that might be refactored soon anyway. will leave it for now, until I added the Entity query